app/src/main/java/it/quip/android/actvitiy/QuipitHomeActivity.java

```diff
@@ -84,8 +86,10 @@ private void updateSidebarMenu() {
         Menu circlesSubMenu = mNavDrawer.getMenu().findItem(R.id.navCircles).getSubMenu();
         circlesSubMenu.clear();
 
-        for (Circle circle : user.getCircles()) {
-            circlesSubMenu.add(0, (int) circle.getUid(), Menu.NONE, circle.getName());
+        List<Circle> userCircles = user.getCircles();
+        for (int i = 0; i < userCircles.size(); i++) {
+            Circle circle = userCircles.get(i);
+            circlesSubMenu.add(0, i, Menu.NONE, circle.getName());
```

Comment:
@jcomo I think this should do the trick now. Use the index of circles in the user circles list as the itemId. I don't think there should be any collisions with the notification ids.